blog/_posts/2019-09-25-scala-contributors-summit.md

```diff
@@ -0,0 +1,79 @@
+
+Around 50 Scala library contributors, authors, and maintainers gathered and discussed various topics, such as the upcoming changes in Scala 3, improvements to the documentation and the website, the future of Scala tooling for both OSS developers and big companies, proposals to better collaborate online and welcome more contributors, etc. Participants shared with us that they learned a lot from each other, how helpful it was to be present in person and meet many of their colleagues, and exchange experience.
+
+No specific program was proposed. Instead, topics and groups were formed on the spot: anyone could propose a topic and interested participants would join for discussion.
+
+Significant result: Scala Improvement Process took place during the Contributors Summit, and public was for the first time present during the meeting in person. In this particular meeting the SIP committee agreed on approaching Scala 3 changes, dividing worklad in batches as well as giving a month per batch to collect community's input. For more on this, please read the [SIP minutes](https://docs.scala-lang.org/sips/minutes/2018-05-18-sip-minutes.html).
+
+Otherwise in this edition, even though some were taking notes, not all were collected, therefore will not be published here. (If you have notes, please share them with us scala.center@epfl.ch)
+
+### Scala Days EPFL Lausanne, 10th June 2019, all day, sponsored by VirtusLab
+
+Great feedback from the Berlin edition, we decided to keep the tradition and organised it a day before, kick-starting the 10th Anniversary of the Scala Days conference in Lausanne, June 2019.
+This time, there were over 60 participants that were discussing, amongst other:
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">It&#39;s the Scala Contributors Summit at <a href="https://twitter.com/scaladays?ref_src=twsrc%5Etfw">@scaladays</a> today, starting with the contributors assembling at the place where it all started <a href="https://t.co/GZcXozC3Ut">pic.twitter.com/GZcXozC3Ut</a></p>&mdash; Stefan Zeiger 🦋 (@StefanZeiger) <a href="https://twitter.com/StefanZeiger/status/1138072098079936514?ref_src=twsrc%5Etfw">June 10, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+| Topic        | Questions           | 
+| ------------- |:-------------:| 
```

Comment:
```suggestion
| ------------- | ------------- | 
```
would align the second column on the left as well, rather than centered. To me it feels easier to read left-aligned, but I might be the only one.